scripts/run_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
It doesn't depend on bash per se. It should work with any POSIX compliant shell installed as /bin/sh. Which is definitely going to be true for any unix like system but probably not for ms-windows. Having said that, sure, I don't see any reason it couldn't be a xonsh script. I'll rework it.